### DISCUSSION TITLE
[docs][guides] improve Sourcing from Netlify CMS doc

### DISCUSSION THREAD

**Discussion by marcysutton** [gatsbyjs/gatsby#31207] [Ideas / Feature Requests]
## Summary

In feedback from gatsbyjs.org, the Sourcing from Netlify CMS doc could use some clarifications:

> "Netlify CMS allows you to have multiple fields containing markdown. I believe this example is just showing a single \"html\" catch-all markdown field."

> "Hi you are missing instructions to add ClientID and Client Secret to Access Control which I guess should be part of the Authenticating with GitHub section?"

It would be nice to address these items as part of https://github.com/gatsbyjs/gatsby/issues/13804. There is also a tutorial in the works for the Additional Tutorials section, in https://github.com/gatsbyjs/gatsby/pull/14945, and an upcoming deployment recipe. This guide is a bit of overlap with that new tutorial (which should be 100% step by step), but that's okay. This one can stick around as a higher-level [reference guide](https://www.gatsbyjs.org/contributing/docs-templates/#reference-guides). 

If anyone wants to pick this up and has questions, let us know!

**Comment by Carike** [comment]
Hi @marcysutton 

I would love to give this a go. I do have a few questions about it since I'm completely new to Gatsby, but if your perhaps available that I can just run stuff by you I would appreciate it.

Looking forward to hearing from you!

**Comment by jonniebigodes** [comment]
@Carike @marcysutton and the gatsby team are away this week(Monday to Friday) for their meetup and Gatsby days London and with that it's going to be a bit difficult for replying and handling some items like this one, do you mind waiting a bit longer on this until the team returns?

Looking forward to see this addition to the documentation added in the near future.

**Comment by Carike** [comment]
@jonniebigodes that's 100% with me.

Thank you very much for the response and all the best for your meetups & Gatsby days!

**Comment by marcysutton** [comment]
Sounds great @Carike, I'd say go for it!

**Comment by nik72619c** [comment]
Shall I work on this if the PR has not been made yet ?

**Comment by lannonbr** [comment]
@nik72619c it looks like @Carike will be working on this so just because there isn't a PR up yet doesn't mean they aren't working on it.

**Comment by nik72619c** [comment]
no issues ! I dont wish to step up before @Carike either.

**Comment by Carike** [comment]
Hi all!

Sorry, I'm still busy with this!

**Comment by marcysutton** [comment]
We'd love to hear more when you have it ready @Carike!

**Comment by marcysutton** [comment]
Hi @Carike, are you still working on this issue? If not, it would help to open it back up to the community. Thanks so much!

**Comment by nik72619c** [comment]
Hi,
Any work done on this issue yet ? 😅

**Comment by wardpeet** [comment]
We're happy to get updates on https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/sourcing-from-netlify-cms.md.
